endpoints/cookie_sync_test.go

```diff
@@ -1756,12 +1859,22 @@ func TestCookieSyncHandleResponse(t *testing.T) {
 		} else {
 			bidderEval = []usersync.BidderEvaluation{}
 		}
-		endpoint.handleResponse(writer, syncTypeFilter, cookie, privacyMacros, test.givenSyncersChosen, bidderEval, test.givenDebug)
+		endpoint.handleResponse(writer, syncTypeFilter, cookie, privacyMacros, test.givenSyncersChosen, bidderEval, test.givenDebug, test.givenSetCookieDeprecationHeader, test.givenCookieDeprecationTTLSec)
 
 		if assert.Equal(t, writer.Code, http.StatusOK, test.description+":http_status") {
 			assert.Equal(t, writer.Header().Get("Content-Type"), "application/json; charset=utf-8", test.description+":http_header")
 			assert.Equal(t, test.expectedJSON, writer.Body.String(), test.description+":http_response")
 		}
+
+		// assert.Equal(t, writer.Header().Get("Set-Cookie"), "receive-cookie-deprecation=1; Path=/; Expires="+time.Now().Add(time.Second*time.Duration(test.givenCookieDeprecationTTLSec)).Format("Mon, 02 Jan 2006 15:04:05 GMT")+"; HttpOnly; Secure; SameSite=None", test.description+":set_cookie_deprecation_header")
```

Comment:
Why is this assert statement commented out?

Reply:
removed